resources/Microsoft.Windows.Setting.Accessibility/Microsoft.Windows.Setting.Accessibility.psm1

```diff
@@ -480,16 +490,16 @@ class TextCursor
         }        
     }
 
-    static [int] GetThickness()
+    static [int] GetKeyboardShortcutStatus()
     {
-        $thicknessArgs = @{ Path = $global:ControlPanelDesktopRegistryPath; Name = ([TextCursor]::ThicknessProperty); }
-        if (-not(DoesRegistryKeyPropertyExist @thicknessArgs))
+        $KeyboardShortcutStatusArgs = @{ Path = $global:ControlPanelDesktopRegistryPath; Name = ([TextCursor]::KeyboardShortcutStatusProperty); }
+        if (-not(DoesRegistryKeyPropertyExist @KeyboardShortcutStatusArgs))
         {
             return 1
         }
         else
         {
-            $textCursorSetting = (Get-ItemProperty @thicknessArgs).CaretWidth
+            $textCursorSetting = (Get-ItemProperty @KeyboardShortcutStatusArgs).CaretWidth

```

Comment:
Why did these get edited?

Reply:
Part of being incomplete is missing the check for unintentional changes. These will be reverted.